SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentTestUtils.h

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Did something go wrong with the merge?

Reply:
Does this file need to be removed? I was having issues removing files yesterday.